hwriter/hwriter.go

```
@@ -29,19 +29,26 @@ func Writer(resp io.ReadCloser, c *app.RequestContext) error {
 		n, err := resp.Read(buf)
 		if err != nil {
 			if err == io.EOF {
-				break // 读取到文件末尾
+				c.Write(buf[:n])
```

golangci-lint (1.64.8)

Error return value of `c.Write` is not checked

(errcheck)